go.mod

```diff
@@ -55,8 +54,8 @@ require (
 	gopkg.in/resty.v1 v1.12.0
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.1 // indirect
-	gorm.io/driver/postgres v1.0.8
-	gorm.io/gorm v1.21.7
+	gorm.io/driver/postgres v1.4.6
```

Comment:
I saw there is a new release: https://github.com/go-gorm/postgres/releases/tag/v1.5.0

Reply:
Yes, but unfortunately it points to a non-tagged commit of gorm as mentioned in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1637#issuecomment-1471500564, which is the reason I am holding this for now

Reply:
Ah right, thanks for the update. I didn't go as far as checking if that's changed with the new release. Thanks for checking.